ca/ca.go

```diff
@@ -34,6 +36,7 @@ type CAImpl struct {
 	log              *log.Logger
 	db               *db.MemoryStore
 	ocspResponderURL string
+	clockSource      clock.Clock
```

Comment:
Here and throughout, let's rename `clockSource` to `clk`, to match [Boulder](https://github.com/letsencrypt/boulder/blob/89b07f4543e067d21e933ec2603e87f399669912/ca/ca.go#L96).